07-modeling.Rmd

```diff
@@ -340,23 +349,24 @@ des_obj %>%
     outcomevar ~ x1 + x2 + x3,
     na.action = na.omit,
     df.resid = degf(.),
-    family = quasibinomial  #use this to avoid warning about non-integers
+    family = quasibinomial
   )
 ```
 
 Note `svyglm()` is the same function used in both ANOVA and linear regression. However, we've added the link function quasibinomial. While we can use the binomial link function, it is recommended to use the quasibinomial as our weights may not be integers, and the quasibinomial also allows for overdispersion. The quasibinomial family has a default logit link which is what is specified in the equations above. When specifying the outcome variable, it will likely be specified in one of two ways with survey data:
 
-  - A factor variable where not being the first level of the factor indicates a "success"
+  - A two level factor variable where the first level of the factor indicates a "failure" and the second level indicates a "success"
   - A numeric variable which is 1 or 0 where 1 indicates a success
   - A logical variable where TRUE indicates a success
 
 ### Example
```

Comment:
Do we want to add another example that includes multiple variables/interactions?  There are specific ways to interpret interactions with logistic regression that could be informative to readers.  Thoughts?